modules/config/config.go

```diff
@@ -182,3 +183,14 @@ func generateConfig() {
 	fmt.Println("默认配置文件已生成，请修改 config.yml 后重新启动!")
 	_, _ = input.ReadString('\n')
 }
+
+func expand(s string, mapping func(string) string) string {
+	r := regexp.MustCompile(`\${(.*?)}`)
```

Comment:
see https://stackoverflow.com/questions/2821043/allowed-characters-in-linux-environment-variable-names